pkg/api/serialization_test.go

```diff
@@ -42,6 +45,82 @@ func fuzzInternalObject(t *testing.T, forVersion string, item runtime.Object, se
 		func(j *authorizationapi.ClusterPolicyBinding, c fuzz.Continue) {
 			j.RoleBindings = make(map[string]*authorizationapi.ClusterRoleBinding)
 		},
+		func(j *authorizationapi.RoleBinding, c fuzz.Continue) {
+			c.FuzzNoCustom(j)
+			for i := range j.Subjects {
+				kinds := []string{authorizationapi.UserKind, authorizationapi.SystemUserKind, authorizationapi.GroupKind, authorizationapi.SystemGroupKind, authorizationapi.SAKind}
+				j.Subjects[i].Kind = kinds[c.Intn(len(kinds))]
+				switch j.Subjects[i].Kind {
+				case authorizationapi.UserKind:
+					j.Subjects[i].Namespace = ""
+					if valid, _ := uservalidation.ValidateUserName(j.Subjects[i].Name, false); !valid {
+						j.Subjects[i].Name = "validusername"
```

Comment:
> should you add the index to the hard-coded names/namespaces so that if you get two, they are distinct?

doesn't matter for fuzzing, but I don't have an objection.